test/simplifyExpression/fractionsSearch/divideByGCD.test.js

```diff
@@ -15,5 +19,19 @@ describe('simplifyFraction', function() {
     ['-3/-2', '3/2'],
     ['-1/-1', '1'],
   ];
-  tests.forEach(t => testdivideByGCD(t[0], t[1]));
+  tests.forEach(t => testDivideByGCD(t[0], t[1]));
+});
+
+describe('simplifyFraction', function() {
+  const tests = [
+    ['15/6',
+     ['15/3 / (6/3)'],
```

Comment:
I think the numerator should have parens around it too (if you keep this version of the rule)